src/Tools/IdeBenchmarks/SQLitePersistentStorageBenchmark.cs

```diff
@@ -127,10 +127,7 @@ private class SolutionSizeTracker : ISolutionSizeTracker
 
         private class LocationService : IPersistentStorageLocationService
         {
-            public event EventHandler<PersistentStorageLocationChangingEventArgs> StorageLocationChanging { add { } remove { } }
-
-            public bool IsSupported(Workspace workspace) => true;
-            public string TryGetStorageLocation(SolutionId solutionId)
+            public string TryGetStorageLocation(Solution _)
             {
                 // Store the db in a different random temp dir.
                 var tempDir = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
```

Comment:
an example of an implementation that doesn't use a workspace with a filepath.  so it provides its own unique location to place hte sqlite db.